oozie-to-airflow/tests/mappers/test_mapreduce_mapper.py

```diff
@@ -20,14 +20,12 @@
 
 from airflow.utils.trigger_rule import TriggerRule
 
-from converter.primitives import Task, Relation
+from converter.primitives import Relation
```

Comment:
Why did you stop using the Task object? In my opinion it is very cool. The representation that shows the IDE is a valid Python code In many cases, you just need to copy it to have a ready test. You can easily test the entire mapper result.

Reply:
I've changed it back to use `Task` in https://github.com/GoogleCloudPlatform/oozie-to-airflow/pull/159/commits/17a8ff7174d603276307db0447315702c89209a0